app/Controllers/authController.php

```diff
@@ -160,7 +160,7 @@ public function formLoginAction() {
 				Minz_Translate::init(FreshRSS_Context::$user_conf->language);
 
 				// All is good, go back to the index.
-				Minz_Request::good(_t('feedback.auth.login.success'), [ 'c' => 'index', 'a' => 'index' ]);
+				Minz_Request::good(_t('feedback.auth.login.success'), Minz_Url::unserialize(Minz_Request::param('original_request')));
```

Comment:
Probable regression: https://github.com/FreshRSS/FreshRSS/issues/4766

Reply:
I had a problem of a white page after first logging in on a brand new install, I had to refresh the page to get it working. Could it be linked? I thought it was my specific setup but I can try to reproduce it.

Reply:
Yes, that's it @Alwaysin

Reply:
@Alwaysin could you check #4778 and tell me if this fixes your issue?

Reply:
@aledeg Yes, it does!